L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -134,14 +134,26 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
     }
 
     func test13ReloadButtonContextMenu() {
+        navigator.performAction(Action.ToggleTrackingProtectionSettingAlwaysOn)
```

Comment:
Tip: use 

```swift
navigator.toggleOn(userState.trackingProtectionSetting == FxUserState.TrackingProtectionSetting.alwaysOn.rawValue, withAction: Action.ToggleTrackingProtectionSettingAlwaysOn)
```

This would make it tolerant to us changing the defaults in the future, but… considerably ugly in the present.